molecule/kvm-single-server/prepare.yml

```diff
@@ -9,5 +9,5 @@
   tasks:
     - name: Update APT package cache
       ansible.builtin.apt:
-        update_cache: true
+        update_cache: "{{ wireguard_update_cache }}"
```

Comment:
I actually what to keep it as is. This a Molecule test and in this phase only the VMs and a little bit of other stuff gets initialized. So at this phase the Wireguard role isn't involved.